game/src/it/scala/ScalatraBootstrapSpec.scala

```diff
@@ -0,0 +1,24 @@
+import io.aigar.controller._
+import io.aigar.controller.response._
+import io.aigar.game._
+
+import org.json4s._
+import org.json4s.jackson.JsonMethods._
+
+import org.scalatra.test.specs2._
+import org.specs2.matcher._
+
+class ScalatraBootstrapSpec extends WholeAppTest {
+  protected implicit val jsonFormats: Formats = DefaultFormats
+
+  "Launching the whole application" should {
+    "launch a game thread that sets the state of the ranked game" in {
+      get("/api/1/game/" + Game.RankedGameId.toString) {
+        status must be_==(200).eventually
```

Comment:
[40 times](https://etorreborre.github.io/specs2/api/SPECS2-3.8.5/index.html#org.specs2.matcher.EventuallyMatchers@eventually[T](nested:=>org.specs2.matcher.Matcher[T]):org.specs2.matcher.Matcher[T]). We could change it/add a sleep between the tries too.